scripts/test

```diff
@@ -40,7 +40,12 @@ copy_logs(){
 
 run_tests()
 {
-    pushd ./tests/validation
+    if [ -n "$TEST_ROOT_DIR" ];
+    then
+        pushd ./tests/validation_v2
+    else
+        pushd ./tests/validation
+    fi
```

Comment:
Instead of the IF statement, lets add this around line 6
: ${TEST_ROOT_DIR:=./tests/validation}
Add TEST_ROOT_DIR to the exports list on line 11.

Then this IF statement reduces down to 
pushd ${TEST_ROOT_DIR}